src/__tests__/seconds-to-time-string.test.js

```diff
@@ -0,0 +1,55 @@
+const { secondsToAge } = require("../utils/seconds-to-age");
```

Comment:
extra bonus northcoders points for using TDD to build this util function! amazing!